app/src/main/java/com/example/roomremix2/MainActivity.kt

```diff
@@ -62,7 +62,6 @@ class MainActivity : AppCompatActivity() {
         }
 
         //decorating part
-//val couchStamp = findViewById<ImageButton>(R.id.couch).drawable
```

Comment:
Agreed.  
Avoid pushing commented code to Version Control.  Version control gives us better options:
- You can look at history to see what used to be there.
- You can use branches for experimental features.